landing-pages/site/layouts/shortcodes/accordion.html

```diff
@@ -25,7 +25,9 @@
         </div>
         <div class="accordion__summary-content">
             {{ if $icon }}
-                {{ with resources.Get $icon }}{{ ( . | minify).Content | safeHTML }}{{ end }}
+                <div class="accordion__summary-content--icon">
+                    {{ with resources.Get $icon }}{{ ( . | minify).Content | safeHTML }}{{ end }}
```

Comment:
Something is wrong here
<img width="1244" alt="Screenshot 2019-10-26 at 14 35 59" src="https://user-images.githubusercontent.com/12058428/67619611-f58dac80-f7fd-11e9-9678-1f25648bb648.png">

Reply:
Hmm that's weird, because it works well on other pages. For some reason everything below icon gets wrapped in `<pre><code>` tags. Maybe we could ignore it since `Example` page gets deleted anyways?

Reply:
I fixed it.